.ci_support/win_64_numpy2.0python3.10.____cpythonpython_implpypy.yaml

```diff
@@ -15,15 +15,15 @@ libcblas:
 liblapack:
 - 3.9 *netlib
 numpy:
-- '1.22'
+- '2.0'
 pin_run_as_build:
   python:
     min_pin: x.x
     max_pin: x.x
 python:
 - 3.10.* *_cpython
 python_impl:
-- cpython
+- pypy
```

Comment:
So I spent the last ~2-3 hours debugging this, made harder by the fact that the rerender changes based on random print statements or other changes I've made to my local smithy checkout (or even the `ordering:` length in the pypy migrator).

I ended up writing my findings in https://github.com/conda-forge/conda-smithy/issues/1917

Reply:
Now using a dev-version of smithy to work around this. If the bot does add it back, it's not a big issue as this value doesn't actually influence anything in the build.